src/net/sourceforge/kolmafia/request/AdventureRequest.java

```diff
@@ -55,6 +56,88 @@
 public class AdventureRequest extends GenericRequest {
   public static final String NOT_IN_A_FIGHT = "Not in a Fight";
 
+  public static final Set<String> ZONELESS_NCS =
```

Comment:
I might want to see these in a datafile. Maybe we could make noncombats.txt to future proof against any other data we want to track about NCs in future. canWalkAway could move there, for example.

Reply:
For example, your comment that mafia should have a list of all superlikelies - that maybe lives there also? Though you can have superlikely combats too. Hm

Reply:
Really combats.txt needs to turn into encounters.txt, with the attributes after the colon for each entry specifying if it is an NC, and indeed if it is/can be superlikely. Something like that.

Reply:
I agree with this 100% - but don't want to expand the scope of this PR too much. It looks like encounters.txt already has some of this. I'll see what I can do to bring those in. I don't know if we want to double down on that design but it does exist.